README.md

```diff
@@ -6,6 +6,26 @@
    pip install poetry && poetry install && poetry run pre-commit install
 ```
 
+## Trufflehog pre-commit hook
+
+- Installation on Mac
+
+```shell
+brew install trufflehog
+```
+
+Alternative installation methods [here](https://github.com/trufflesecurity/trufflehog)
+
+
+#### Running Trufflehog via Docker
+
+To run trufflehog in a Docker container there is a ```run-trufflehog.sh``` script which handles the Docker command and ensures your project directory is correctly mounted inside the container. The working directory is set to the current working directory.
```

Comment:
Probably just get rid of the script and these instructions for the same reason as the alternative installation. The hooks won't work with the script without additional messing around (renaming the script and putting it on the path, or hacking the hook which shouldn't be encouraged)

Reply:
ah i see, no worries will do